python-stubs/dolphin/controller.pyi

```diff
@@ -4,9 +4,10 @@ Module for programmatic inputs.
 Currently, only for GameCube, Wiimote buttons and Wii IR (pointing).
 No acceleration or extensions data yet.
 """
-from typing import TypedDict
+from typing import TypedDict, type_check_only
 
 
+@type_check_only
```

Comment:
If you could, these would be beneficial to have during runtime, otherwise you have to use them as such to use them as type annotations:
![image](https://user-images.githubusercontent.com/1350584/222532310-c4d8320c-3ce5-402c-976b-0d6c55a9d083.png)

See below, used directly is fine, as it is inferred as the expected parameter type, but inferred as `dict[str, bool]` is not:
![image](https://user-images.githubusercontent.com/1350584/222533039-f3810781-0c09-40bc-83ad-3b106ee39cd1.png)